### ISSUE TITLE
Space Station Silicon Valley colors from texture not being picked correctly

### ISSUE THREAD

**Issue by JappaWakkaP3D** [gonetz/GLideN64#2457]
In Space Station Silicon Valley, Dan Danger appears to be fully pink. The game should pick the colors from the colored squares at the top, but when using a hires texture, for some reason the game doesn't do that.
![image](https://user-images.githubusercontent.com/66885565/107855172-1e85e680-6e21-11eb-8e90-dcb7d52ed1de.png)
This is the original texture:
![Silicon Valley#888B8774#0#2_all](https://user-images.githubusercontent.com/66885565/107855293-a4099680-6e21-11eb-8b13-5f2f532012b6.png)

and this is mine:
![Silicon Valley#888B8774#0#2_all](https://user-images.githubusercontent.com/66885565/107855300-b257b280-6e21-11eb-9b28-d539003f0c3a.png)

Am I doing something wrong?

**Comment by Jj0YzL5nvJ** [comment]
<details>
  <summary>sha256sum 107855*</summary>

```
64baefaf274776f4c97f2c4d72f20bdc6b24593d69aae66bbfb7efc3689f54a9  107855293-a4099680-6e21-11eb-8b13-5f2f532012b6.png
de41d0bca73ef3dfce6fa4e135bf148f0b6652f544da3a18a08bc6cdff18586b  107855300-b257b280-6e21-11eb-9b28-d539003f0c3a.png
```

</details>

<details>
  <summary>mediainfo 107855*</summary>

```
General
Complete name                            : 107855293-a4099680-6e21-11eb-8b13-5f2f532012b6.png
Format                                   : PNG
Format/Info                              : Portable Network Graphic
File size                                : 724 Bytes

Image
Format                                   : PNG
Format/Info                              : Portable Network Graphic
Width                                    : 32 pixels
Height                                   : 32 pixels
Bit depth                                : 32 bits
Compression mode                         : Lossless
Stream size                              : 724 Bytes (100%)

General
Complete name                            : 107855300-b257b280-6e21-11eb-9b28-d539003f0c3a.png
Format                                   : PNG
Format/Info                              : Portable Network Graphic
File size                                : 5.31 KiB

Image
Format                                   : PNG
Format/Info                              : Portable Network Graphic
Width                                    : 256 pixels
Height                                   : 256 pixels
Bit depth                                : 24 bits
Compression mode                         : Lossless
Stream size                              : 5.31 KiB (100%)
```

</details>

#2287

**Comment by JappaWakkaP3D** [comment]
I changed the bit depth but it's still not working

**Comment by JappaWakka** [comment]
![image](https://user-images.githubusercontent.com/31563291/108052682-c7fcf000-704c-11eb-85dd-b73282f86390.png)
Changing the texture size back to 32x32px seems to fix it strangely enough

**Comment by Jj0YzL5nvJ** [comment]
Mmm... texture as bitmap?

**Comment by JappaWakkaP3D** [comment]
Texture as bitmap? I don't know what you mean?

**Comment by Jj0YzL5nvJ** [comment]
Maybe the game uses specific coordinates of the image as a color palette...

**Comment by JappaWakka** [comment]
yeah definitely but I feel like the coordinates are incorrect. if I make the color palette at the top 1:1 with the original, everything is pink, even the things that weren't before.

**Comment by JappaWakka** [comment]
![image](https://user-images.githubusercontent.com/31563291/108055618-83735380-7050-11eb-8a08-f4863a391af6.png)
![Silicon Valley#888B8774#0#2_all](https://user-images.githubusercontent.com/31563291/108055656-8ec67f00-7050-11eb-9b17-a48b3087e051.png)
this is what it looks like when I offset the colors like this

**Comment by Jj0YzL5nvJ** [comment]
[Try this version.](https://ci.appveyor.com/project/gonetz/gliden64/builds/37752901/artifacts)

**Comment by JappaWakka** [comment]
same problem

**Comment by Jj0YzL5nvJ** [comment]
Can you confirm whether Rice produces the expected result?

**Comment by JappaWakka** [comment]
Rice doesn't use the same names right?

**Comment by JappaWakka** [comment]
![image](https://user-images.githubusercontent.com/31563291/108071030-82005600-7065-11eb-9181-a8baca914859.png)
![Silicon Valley#ACE58699#0#2_all](https://user-images.githubusercontent.com/31563291/108071058-8b89be00-7065-11eb-9a5c-adf851de5d13.png)
almost got it. it seems to take some colors from the top and others from the bottom of the texture, for example: the white block at the bottom is his glasses, but the one at the top is his teeth. I just gotta mess around a bit with the colors

**Comment by Jj0YzL5nvJ** [comment]
> Rice doesn't use the same names right?

Rice support with this game on is really bad...

Can you try this one?

![Silicon Valley#888B8774#0#2_all](https://user-images.githubusercontent.com/12230570/108119754-77da5980-705d-11eb-8ea9-088ef0ad9362.png)
